### PR TITLE
generic orgs

### PR DESCRIPTION
**Pull request by nbittich**
OP-2787 frontend part. this is not complete as I would suggest we should maybe rename everything named with "administrative-unit-***" by something more generic

### CODE REVIEW

File: app/templates/administrative-units/index.hbs [hunk 1]
```diff
@@ -167,13 +167,18 @@
...
-              </LinkTo>
+
+              </AuLink>
+              {{! <LinkTo }}
+              {{!   class="au-c-link" }}
+              {{!--   @model={{adminUnit.id}} --}}
+              {{!--   @route="{{(fn getOrgRoute adminUnit)}}" --}}
+              {{! > }}
+              {{!--   {{adminUnit.name}} --}}
+              {{! </LinkTo> }}
```
Comment (claire-lovisa) [inline app/templates/administrative-units/index.hbs:181]: This can be removed ?

**Review by claire-lovisa — APPROVED** [review]
Working as it is ! But indeed the refactoring cleanup will be much appreciated. We also will be able to merge the `bestuuresenheden` and `organisaties` modules right ?

**Comment by nbittich** [comment]
@claire-lovisa ticket OP-2796 is for the full code refactoring, I'd suggest to wait for ember 4 otherwise it will be a mess 😬 
We probably want to merge module administrative-units & organizations in the code if that's what your meant 🤔

**Comment by claire-lovisa** [comment]
> @claire-lovisa ticket OP-2796 is for the full code refactoring, I'd suggest to wait for ember 4 otherwise it will be a mess 😬 We probably want to merge module administrative-units & organizations in the code if that's what your meant 🤔

Indeed I meant merge as : those two module will be able to become just one !

**Review by claire-lovisa — APPROVED** [review]

**Pull request merged**